sale_order_secondary_unit/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,4 @@
+To configure this module, you need to:
+
+#. Go to *Sales > Configuration > Products > Secondary Units of Measure*
```

Comment:
This menu is activated only when `sale_management` is installed (https://github.com/odoo/odoo/blob/e78f558b2c09c8616b50b92778a06bc9cbb5ae66/addons/sale_management/views/sale_management_menus.xml#L5), maybe we should depend on that one?
Or maybe just add a note here about that.